Sources/MapboxCoreNavigation/LegacyRouteController.swift

```diff
@@ -93,12 +95,14 @@ open class LegacyRouteController: NSObject, Router, InternalRouter, CLLocationMa
         UIDevice.current.isBatteryMonitoringEnabled = true
 
         super.init()
-        
+        BillingHandler.shared.beginBillingSession(for: .activeGuidance, uuid: sessionUUID)
```

Comment:
Do we need a disclaimer at the end of this file about billing related code too?

Reply:
Good catch!